src/Components/Todo/index.jsx

```diff
@@ -73,29 +75,40 @@ const Todo = () => {
   const [incomplete, setIncomplete] = useState([]);
   const { handleChange, handleSubmit } = useForm(addItem, defaultValues);
 
-  function addItem(item) {
-    item.id = uuid();
+  async function addItem(item) {
     item.complete = false;
-    console.log(item);
+    const config = {
+      baseURL: 'https://api-js401.herokuapp.com/api/v1/todo',
+      method: 'post',
+      data: item
+    };
+    let response = await axios(config);
+    console.log('item', response.data);
     setList([...list, item]);
   }
```

Comment:
Make sure you are adding the item object you get back from the server response, because we need the `item._id` that the server generates. Right now we can't update or delete until we refresh the page.
```js
setList([...list, response.data]);